src/main/java/com/book/booksstore/service/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.book.booksstore.service;
+
+import com.book.booksstore.model.Book;
+import com.book.booksstore.repository.BookRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class BookServiceImpl implements BookService {
+
+    private final BookRepository bookRepo;
```

Comment:
```suggestion
    private final BookRepository bookRepository;
```
Don’t  use abbreviations

Reply:
done - renamed
![image](https://github.com/user-attachments/assets/6ed1d1a9-27ad-492c-b788-6e3627966b90)